spec/registry.xml

```diff
@@ -6,6 +6,7 @@
   <validationRule id="positiveInteger" regex="0|([1-9]\d*)"/>
   <validationRule id="currencyCode" regex="[A-Z]{3}"/>
   <validationRule id="timeZoneId" regex="[a-zA-Z/]+"/>
+  <validationRule id="anything" regex=".*"/>
```

Comment:
Technically this should be the same as a post-processed literal, which would omit surrogates. Probably not worth dinking with?

Reply:
That's taken care of by the syntax rule, so a literal value will never include surrogates.